src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_20_CertificateReissuance.swift

```diff
@@ -4,7 +4,7 @@
 
 import XCTest
 
-class ENAUITests_CertificateReissuance: CWATestCase {
+class ENAUITests_20_CertificateReissuance: CWATestCase {
```

Comment:
👍

Reply:
Just out of curiosity, why is the number needed?

Reply:
More like a sequence in which the features were built.

Reply:
And we already have this pattern, why breaking it?

Reply:
Because its not needed.

Reply:
I created a topic for our dev camp to discuss this there.